package.json

```diff
@@ -39,6 +39,7 @@
       "@types/node"
     ]
   },
+  "name": "sonarwhal-meta",
```

Comment:
I need a name to find if we are creating an official rule or not.

Reply:
`@sonarwhal/monorepo`?